test/extended/deployments/deployments.go

```diff
@@ -1,6 +1,7 @@
 package deployments
 
 import (
+	//"errors"
```

Comment:
There are also commented parts of code which use it this is because this PR was split in half and the next one uncomments it. Also felt good to leave it because a had to rename kubernetes errors package to kerrors because of it